src/components/forms/index.tsx

```diff
@@ -1 +1 @@
-export { default as ContactForm } from "./ContactForm";
+export { default as ContactForm } from "./ContactForm/ContactForm";
```

Comment:
What do you think about removing this index.tsx as well?

Reply:
I think using index files simplifies imports, hides folder structure details, and centralizes exports, making the codebase cleaner, easier to manage, and more scalable.